src/main/java/org/opensearch/searchrelevance/model/Experiment.java

```diff
@@ -33,7 +33,8 @@ public class Experiment implements ToXContentObject {
     private static final int DEFAULTED_SIZE = 10;
 
     /**
-     * Identifier of the system index
+     * Identifier of the system index will be equivalent to any
```

Comment:
I think we don't need this change on Experiment.java.   I think the comment on ScheduledJob was good, but this one we should back out.   And honeslty, I think we don't even need the comment, I don't think it helps elabored.  I might just ditch the whole comment on the id field.  we don't comment every field, and we know what id is...